app/Console/Commands/RussianRoulette.php

```diff
@@ -0,0 +1,376 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Contracts\Cache\Repository as CacheRepo;
+
+class RussianRoulette extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'play:roulette';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = "Play the russian roulette game.";
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    private CacheRepo $register;
+    private $players = array('The Computer' => 0, 'You' => 0);
```

Comment:
Why is used old php array declaration?

Reply:
It has no purpose, i just used this way accidentally.